### ISSUE TITLE
High percentage of passwords start with 'A'

### ISSUE THREAD

**Issue by daemonblade** [eterevsky/passwordgen#9]
This seems a bit off to me.

Out of the 25 passwords that I have, 16 (ie 64%) of them start with 'A'. Is this to be expected? I somehow expected a more even distribution of characters.

**Comment by eterevsky** [comment]
Yep, I'm investigating. Seems like an issue of conversion from hash to string.

**Comment by pedrosimao** [comment]
Has the issue been solved. It's been 5 years + since it was open.
Is there a real issue with the algo generating the passwords?
Does it create security exploits?
